solr/solr-ref-guide/src/major-changes-in-solr-9.adoc

```diff
@@ -53,7 +53,11 @@ SolrCloud doesn't support that but may sometime.
 
 * SOLR-14702: All references to "master" and "slave" were replaced in the code with "leader" and "follower".
 This includes API calls for the replication handler and metrics.
-For rolling upgrades into 9.0, you need to be on Solr version 8.7 or greater. Some metrics also changed, alerts and  monitors on Solr KPIs that mention "master" or "slave" will also now be "leader" and "follower"
+For rolling upgrades into 9.0, you need to be on Solr version 8.7 or greater. Some metrics also changed, alerts and monitors on Solr KPIs that mention "master" or "slave" will also now be "leader" and "follower"
+
+* SOLR-15965: Internode communication secured by PKI Authentication has changed formats. To maintain compatability, you will need to either:
+** Upgrade the cluster to 8.11.2 or newer before proceeding to 9.0
```

Comment:
I'm guessing we'll want 8.11.2 to default to `solr.pki.sendVersion=v1` and `solr.pki.acceptVersions=v1,v2` to support rolling upgrades from versions <= 8.11.1, right? if that's the case, even upgrading from 8.11.2 may require setting `solr.pki.acceptVersions=v1,v2` unless the user has already upgraded to v2 explicitly.

Reply:
I tried making the upgrade instructions more explicit. Let me know what you think.